src/main/docker/docker-compose.yml

```diff
@@ -106,7 +106,7 @@ networks:
       driver: default
       config:
         # Configure the IP Address range to not conflict with Deutsche Bahn Wi-Fi
-        - subnet: 172.72.0.1/24
```

Comment:
This leads to the error:

```
Network cyface-collector  Error
failed to create network cyface-collector: Error response from daemon: invalid network config:
invalid subnet 172.72.0.1/24: it should be 172.72.0.0/24
```

I guess the newer versions of docker introduced this subnet definition check.